### ISSUE TITLE
[Bug]: EIP1193Provider incompatible with ethers ExternalProvider

### ISSUE THREAD

**Issue by tuler** [blocknative/web3-onboard#865]
### Expected Behavior

I was expecting the `wallet.provider`, which is a `EIP1193Provider` to be compatible with [ethers ExternalProvider](https://docs.ethers.io/v5/api/providers/other/#Web3Provider--ExternalProvider), but I get the following error:

### Onboard Version

2.0.4

### Node Version

16

### Relevant log output

```shell
Argument of type 'EIP1193Provider' is not assignable to parameter of type 'ExternalProvider | JsonRpcFetchFunc'.
  Type 'EIP1193Provider' is not assignable to type 'ExternalProvider'.
    Types of property 'request' are incompatible.
      Type '{ (args: EthAccountsRequest): Promise<ProviderAccounts>; (args: EthBalanceRequest): Promise<string>; (args: EIP1102Request): Promise<...>; (args: SelectAccountsRequest): Promise<...>; (args: EIP3326Request): Promise<...>; (args: EIP3085Request): Promise<...>; (args: EthChainIdRequest): Promise<...>; (args: EthSignTr...' is not assignable to type '(request: { method: string; params?: any[] | undefined; }) => Promise<any>'.
        Types of parameters 'args' and 'request' are incompatible.
          Type '{ method: string; params?: any[] | undefined; }' is not assignable to type 'EthAccountsRequest'.
            Types of property 'method' are incompatible.
              Type 'string' is not assignable to type '"eth_accounts"'.ts(2345)
```
```


### Anything else?

`"ethers": "^5.5.4"`

**Comment by aaronbarnardsound** [comment]
Going to take a look at this today and see if we can update the types to be compatible. If you ignore the compiler did you find that the functionality was working correctly?

**Comment by aaronbarnardsound** [comment]
Closed via #869
